.gitignore

```diff
@@ -0,0 +1,7 @@
+.gradle
```

Comment:
Seems gitignore was rewrited by old version. Please, change to the github version for android.